### DISCUSSION TITLE
Some markdown content is converted to css when tailwind plugin is used

### DISCUSSION THREAD

**Discussion by QQCif** [tailwindlabs/tailwindcss#10874] [Help] [answered]
Copied from original issue in astro repo withastro/astro#6663
I recently exported a WordPress site to markdown and noticed that some of the content, specifically [a:b], is being converted to css when using Tailwind. I have created an example file, issue.md, for reference. Could you please take a look? Thank you.

**Comment by wongjn** [comment, marked as answer]
The `[a:b]` string matches [Tailwind's arbitrary property](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) class name syntax. You could:
- Remove the markdown files from being scanned from Tailwind if they don't have Tailwind classes within them.
- Use the [`blocklist`](https://tailwindcss.com/docs/content-configuration#discarding-classes) option to discard false positives.
